### PR TITLE
Methap/new filter

### PR DESCRIPTION
**Pull request by mehtapy**

### CODE REVIEW

File: client/src/components/UI/searchbar/SearchJobs.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//       }
+//       if (!isJob(item) && item.company_name && item.company_name.toLowerCase().includes(query.toLowerCase())) {
+//         return true;
+//       }
+//       return false;
+//     });
+
+//     onSearch(result);
+//   };
+
```
Comment (gitadventurer241) [inline client/src/components/UI/searchbar/SearchJobs.tsx:41]: don't leave commented code please :D

Comment (mehtapy) [reply client/src/components/UI/searchbar/SearchJobs.tsx:41]: im not sure if i need until it works :)

File: client/src/components/pages/jobs/FakeDataForJobs.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    employment_type: "Full-time",
+    skills: [
+      { skill_id: "1", skill_name: "JavaScript" },
+      { skill_id: "2", skill_name: "React" },
+      { skill_id: "3", skill_name: "Node.js" },
+    ],
+    soft_skills: ["Teamwork", "Problem Solving"],
+    matchScore: 75,
+    isSaved: false,
+    associations: ["Association 1", "Association 2"],
```
Comment (gitadventurer241) [inline client/src/components/pages/jobs/FakeDataForJobs.tsx:18]: why this file is committed?

**Review by gitadventurer241 — CHANGES_REQUESTED** [review]

File: client/src/components/pages/jobs/FakeDataForJobs.tsx (hunk 2 above)
Comment (mehtapy) [reply client/src/components/pages/jobs/FakeDataForJobs.tsx:18]: I needed it for testing it shouldn’t be there but I did git add . therefore everything is pushed

Comment (gitadventurer241) [reply client/src/components/pages/jobs/FakeDataForJobs.tsx:18]: then please delete it

**Review by albarkauss — APPROVED** [review]

**Pull request merged**